.github/workflows/stale-prs.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow warns and then closes PRs that have had no activity for 90 days.
+#
+# For more information, see:
+# https://github.com/actions/stale
+name: Close stale pull requests
+
+on:
+  schedule:
+  - cron: '0 18 * * *' # Run the workflow every day at 6PM UTC (10AM PST).
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: 'This pull request has been automatically marked as stale because it has been inactive for 90 days. Remove stale label or comment or this PR will be closed in 7 days.'
```

Comment:
Is there a way to opt-out of this behavior? What if we have a PR we know is on hold and we don't want it automatically closed? Maybe we can add a label to indicate that it shouldn't go stale?

Reply:
(although 3 months is a long time so removing the label won't be a daily thing. I think as designed this looks good)

Reply:
Yeah there is an option to opt out for specific labels https://github.com/actions/stale?tab=readme-ov-file#exempt-pr-labels, but I thought we should introduce it if really see the need for it.